org.ekstep.questionunit-1.0/renderer/plugin.js

```diff
@@ -1,6 +1,16 @@
 org.ekstep.contentrenderer.questionUnitPlugin = Plugin.extend({
   _type: 'org.ekstep.questionUnitPlugin',
   _render: true,
+  _addUrduSupport: function(){
+    // Add to Question Title
+    var urduSupportClass = 'urdu-right-align'
+    $('.question-title-text-common-questionunit-plugins').addClass(urduSupportClass);
```

Comment:
Make it a small name. It is too long.
question-title-text-common-questionunit-plugins => common-qs-title

Reply:
Done changes
https://github.com/project-sunbird/sunbird-content-plugins/pull/139/